VAMobile/src/components/Templates/HeaderBanner.tsx

```diff
@@ -110,7 +110,7 @@ const HeaderBanner: FC<HeaderBannerProps> = ({
       updateVaOpacity()
       updateTitleShowing()
     }
-  })
+  }, [transition, title, titleShowing])
```

Comment:
this was a bug that popped up as I was changing things that is a RN bug in particular with a debugger on. So it fixes it as otherwise with demo mode and a debugger running after navigating to upload photos and going back it would crash